relay-server/src/services/processor.rs

```diff
@@ -2101,7 +2102,7 @@ impl EnvelopeProcessorService {
 
     fn handle_process_envelope(&self, message: ProcessEnvelope) {
         let project_key = message.envelope.envelope().meta().public_key();
-        let wait_time = message.envelope.start_time().elapsed();
+        let wait_time = message.envelope.elapsed();
```

Comment:
For all collected metrics (`event.wait_time` and `event.total_time`), we'll have to decide at a later point whether we _also_ want to log a metric for the time since the envelope was dequeued from the buffer.